src/FileIO.chpl

```diff
@@ -308,18 +308,12 @@ module FileIO {
 
     proc lsAnyMsg(cmd: string, payload: string, st: borrowed SymTab): MsgTuple throws {
       var (jsonfile) = payload.splitMsgToTuple(1);
```

Comment:
we shouldn't need the `splitMsgToTuple` line any longer

Reply:
Removed.